ui/apps/ui/src/app/components/right-menu/right-menu.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ess-right-menu',
+  template: `<div>
+    <a [queryParams]="{ q: '*' }" [routerLink]="'/search/provider'"
+      >Providers -></a
+    >
+  </div>`,
+  styles: [``],
```

Comment:
If component does not have styles this part can be skipped

Reply:
styles added